### ISSUE TITLE
VGG pre-trained model

### ISSUE THREAD

**Issue by tinazoe1997** [JDAI-CV/FADA#19]
Hi, thank you for your awesome work! Would you mind releasing the VGG pre-trained weights with and without the self-distillation step? By the way, the FADA mIoU 43.8 shown in the paper is the VGG model with the self-distillation step right?

Best regards,
Ut

**Comment by krumo** [comment]
Thanks for your interest! Yes, without additional explanation, the performance reported in the paper is obtained by the model trained with self-distillation step by default. Unfortunately we don't have a plan to release the VGG models for this project right now. The VGG model results could be reproduced with this repo without any additional tricks. Please let me know if you are faced with any difficulties in reproducing VGG model results.

Best wishes!

**Comment by tinazoe1997** [comment]
@krumo Thanks for your reply!  I was wondering about the performance which is trained without the self-distillation step on vgg models because I got mIoU 39.7 after the self-distillation step(even worse than mIoU 40.17 which I pre-trained FADA by myself)  cannot reach mIoU 43.8. All of the training settings are the same as this repo, except the batch size (8->2) due to the computational limitation.

Best Regards,
Ut

**Comment by pangtina123** [comment]
I have the same question too. What is the performance of the vgg backbone without the self-distillation step? Because the training tricks (especially the self-distillation step) you used usually can bring more boost to the performance.

**Comment by krumo** [comment]
If you would like to use a smaller batchsize for training, I would suggest you adopt the linear scaling rule depicted in paper [
Accurate, Large Minibatch SGD: Training ImageNet in 1 Hour](https://arxiv.org/pdf/1706.02677.pdf) to decrease learning rate and increase training iterations. Before self-distillation step, I could obtain 41.1 mIoU with a VGG backbone on GTA5 to Cityscapes task.

**Comment by tinazoe1997** [comment]
Thanks for your reply! It's very helpful to me!
